src/Symfony/Component/HttpFoundation/Tests/RequestTest.php

```diff
@@ -244,6 +244,21 @@ public function testCreate()
         // Fragment should not be included in the URI
         $request = Request::create('http://test.com/foo#bar');
         $this->assertEquals('http://test.com/foo', $request->getUri());
+
+        // assume no rewrite rule: /index.php --> /
+        $request = Request::create('http://test.com/index.php.php', 'GET', [], [], [],
```

Comment:
Can you make this a new test method instead of adding one more test bundled in the same method (which is then a single test as far as PHPUnit is concerned) ? The HttpFoundation component is very old, and its tests were written in the early days of using PHPUnit in the Symfony community. Because of that, they are not following the PHPUnit best practices in a bunch of places. It would be great if new tests being added could follow them.

Reply:
👍